### DISCUSSION TITLE
Don't switch mode if sharing screen/presenting/duplicating display

### DISCUSSION THREAD

**Discussion by MlleBae** [AutoDarkMode/Windows-Auto-Night-Mode#833] [Q&A]
First of all, THANK YOU so much for this app; this is one of the first apps I download on every PC.

My question is regarding automatically switching modes on my work PC. When I'm in a virtual meeting and sharing my screen or an app, it's very awkward when my PC automatically switches to Dark Mode. It tends to distract from the content of the meeting, and some of my colleagues have a harder time following demos in Dark Mode. 

I looked at the 'Don't switch while certain processes are running' setting, but none of the Processes available seem to target only when I'm sharing my screen. My company uses MS Teams for meetings (I'm currently on 'New Teams' version 24033.813.2773.520). I would be happy to just postpone the switch if any Teams meeting is running, but can't see a process to target this either. Since the Teams app is always running in the background, preventing switching while that process is running would mean never switching. 
I've noticed that Windows makes some settings conditional if you are presenting/sharing your screen ('duplicating your display')–is there a Windows flag/event that could be referenced to identify if presenting is in progress?
![2024-03-29_11-25-39](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/23529588/59d3f9c3-209b-4dde-b2c7-1a0b3314ba27)

Just curious if others have this issue, and if there's already a script or other workaround for this?
Thank you!

**Comment by Jay-o-Way** [comment]
Hi. That's a fair question. I don't really use exclusions like that myself, but it should be possible to add `teams.exe` to the list. Check if that is the executable's file name (try and find a shortcut to it) and type it in. Also, we have been working on a new UI, it's out in beta now - If you would like to try that. Let us know if you found a solution! 🙂
